presto-spark-base/src/main/java/com/facebook/presto/spark/execution/NativeExecutionProcessFactory.java

```diff
@@ -38,10 +38,8 @@
 
 public class NativeExecutionProcessFactory
 {
-    // TODO add config
-    private static final int MAX_THREADS = 1000;
     private static final Duration MAX_ERROR_DURATION = new Duration(2, TimeUnit.MINUTES);
-
+    public static final URI DEFAULT_URI = URI.create("http://127.0.0.1");
```

Comment:
Should this not include a port?

Reply:
The port is selected dynamically by the `NativeExecutionProcess`/`DetachedNativeExecutionProcess` so we don't need to specify it here.